013-quarkus-oidc-restclient/src/test/java/io/quarkus/qe/AbstractPingPongResourceTest.java

```diff
@@ -25,6 +29,14 @@
 
     AuthzClient authzClient;
 
+    @BeforeEach
+    public void setup() {
+        RestAssured.config = RestAssuredConfig.config()
+                .httpClient(HttpClientConfig.httpClientConfig()
```

Comment:
`HttpClientConfig` -> move on to a static import in order to don't have `HttpClientConfig.httpClientConfig()`, sound redundant.

Reply:
Updated